tech/languages/dotnet/about.md

```diff
@@ -0,0 +1,35 @@
+---
+title: .NET
+subsection: dotnet
+section: tech-languages
+order: 1
+version: 7.8.4
+description: ".NET Core and Mono are open-source .NET platforms. .NET applications are developed using the C#, F# and VB.NET programming languages."
+---
+
+.NET Core and Mono are available on Fedora. .NET applications are developed using the C#, F# and VB.NET programming languages.
+
+## .NET Core
+
+[.NET Core](https://docs.microsoft.com/en-us/dotnet/core/) is a general-purpose, modular, cross-platform and open-source development Platform.
+
+To install .NET Core, add the .NET SIG's [copr](http://developer.fedorainfracloud.org/deployment/copr/about.html) repository and install the `dotnet` package.
+
+```
+$ sudo dnf copr enable -y @dotnet-sig/dotnet
+$ sudo dnf install dotnet
+```
+
+See the [.NET Core](dotnet.md) page for more information.
+
+## Mono
+
+[Mono](http://www.mono-project.com/) is a free and open source implementation of the .NET Framework.
+
+To install Mono, simply type:
+
+```
+$ sudo dnf install mono-devel
```

Comment:
Same here: Could you just link to the page below?

Reply:
We want to have the installation instructions on the landing page, since this is the most likely what people are looking for when going to this website.

Reply:
> Why would you install mono-devel and not just mono?

I am not a mono user, I kept the installation instructions as they were. Taking a guess, mono-devel includes the things needed for development.

Reply:
> I am not a mono user, I kept the installation instructions as they were. Taking a guess, mono-devel includes the things needed for development.

Yes, sure. I does actually make sense, in retrospective, as people wan't to develop in that language.

> We want to have the installation instructions on the landing page, since this is the most likely what people are looking for when going to this website.

Well, I really do not know about that, but since the resulting page is short, I do not mind much. Still if you can make it into two lines - two links to subpages - it would make sense to me. (In favor of deduplication.)